RFPs/DRAFT-rfp-013-cryptonet-network-grant.md

```diff
@@ -0,0 +1,46 @@
+# RFP-013: Cryptonet Network Grant
```

Comment:
Are we calling them network grants or microgrants? The cryptonet website says the latter. I prefer the former, given these aren't microgrants (e.g. per FIL's definition, up to $5k).

@rosariogennaro ?

Reply:
Also still pending, this passage in the Cryptonet website:
>An exception to the budget limit will be made for postdoc applications.

Reply:
Reverted to Network Grants and removed the mention of "exception". Cryptonet Website will be updated